src/SFA.DAS.Reservations.Web/Stubs/EmployerAccountApiClientStub.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.EAS.Account.Api.Types;
+using SFA.DAS.Encoding;
+
+namespace SFA.DAS.Reservations.Web.Stubs
+{
+    public class EmployerAccountApiClientStub : IAccountApiClient
+    {
+        private readonly IEncodingService _encodingService;
+        private string _userId;
+
+        public EmployerAccountApiClientStub(IEncodingService encodingService)
+        {
+            _encodingService = encodingService;
+        }
+
+        public Task<ICollection<TeamMemberViewModel>> GetAccountUsers(string accountId)
+        {
+            return GetAccountUsers(1);
+        }
+
+        public Task<ICollection<TeamMemberViewModel>> GetAccountUsers(long accountId)
+        {
+            ICollection<TeamMemberViewModel> teamMembers = new List<TeamMemberViewModel>();
+
+            teamMembers.Add(new TeamMemberViewModel
+            {
+                CanReceiveNotifications = true,
+                Email = "stubby@stubsrus.com",
+                Name = "Stubby McStubface",
+                Role = "Owner",
+                Status = InvitationStatus.Accepted,
+                UserRef = _userId
+            });
+
+            return Task.FromResult(teamMembers);
+        }
+
+        public Task<ICollection<TransferConnectionViewModel>> GetTransferConnections(string accountHashedId)
+        {
+            ICollection<TransferConnectionViewModel> transferConnections = new List<TransferConnectionViewModel>();
+
+            transferConnections.Add(new TransferConnectionViewModel
+            {
+                FundingEmployerAccountId = 456,
+                FundingEmployerAccountName = "Stubs Funding",
+                FundingEmployerHashedAccountId = _encodingService.Encode(456, EncodingType.AccountId),
+                FundingEmployerPublicHashedAccountId = _encodingService.Encode(456, EncodingType.PublicAccountId)
+            });
+
+            return Task.FromResult(transferConnections);
+        }
+
+        public Task<ICollection<AccountDetailViewModel>> GetUserAccounts(string userId)
+        {
+            _userId = userId;
+
+            ICollection<AccountDetailViewModel> accountDetails = new List<AccountDetailViewModel>
+            {
+                new AccountDetailViewModel
+                {
+                    AccountId = 123,
```

Comment:
Same as above, you should put that 123 value into a const so it can be reference elsewhere if needed